app/client/src/features/accounts/ForgotPasswordForm.tsx

```diff
@@ -0,0 +1,13 @@
+import { graphql } from 'react-relay';
+
+const FORGOT_PASSWORD_FORM_MUTATION = graphql`
+    mutation ForgotPasswordFormMutation($input: ForgotPassRequestForm!) {
+        requestResetPassword(input: $input) {
+            isError
+            message
+            body
+        }
+    }
+`;
+
+export { FORGOT_PASSWORD_FORM_MUTATION }
```

Comment:
I think it would be best to merge `ForgotPassRequestForm` and `ForgotPasswordForm` into `ForgotPasswordForm`.  There's not anything wrong with what you did -- it's more so for consistency sake.